src/Microsoft.DotNet.MSBuildSdkResolver/NETCoreSdkResolver.cs

```diff
@@ -20,6 +20,12 @@ public sealed class Result
             /// </summary>
             public string GlobalJsonPath;
 
+            /// <summary>
+            /// True if either (A) there was no applicable global.json or (B) existing global.json was used in resolution. 
+            /// False if a global.json was found but there was no compatible SDK, so it was ignored. 
+            /// </summary>
+            public bool GlobalJsonHonored;
```

Comment:
Maybe call it "FailedToResolveSpecifiedInGlobalJson" and inverse the boolean. So, everything else is true. Currently it is not straght forward to know GlobalJsonHonored == "True if either (A) there was no applicable global.json or (B) existing global.json was used in resolution. "